src/components/SummaryPageCard/SummaryPageCard.styled.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import theme from "../../utils/theme.js";
+
+const sidePaddings = {
+	lg: 40,
+	md: 16,
+}
+
+export const RecentActivityWrapper = styled.div`
+	border: 2px solid ${theme.colors.gray200};
+	font-family: ${theme.fontFamily};
+	width: 100%;
+`;
+
+export const TitleBar = styled.div`
+	align-items: center;
+  background: ${theme.colors.gray200};
+	display: flex;
+	height: 56px;
+	justify-content: space-between;
+	padding: 0 ${(pr) => sidePaddings[pr.size]}px;
+`;
+
+export const TitleWrapper = styled.div`
+	display: flex;
+	align-items: center;
+`;
+
+export const RecentActivityTitle = styled.span`
```

Comment:
This and the "SideCardActionsTitle" shouldn't be created nor exported here... they should come from whoever is invoking the Card component, if devs wanna pass an `<h1>` or an `<a>` or a `<button>` even, they could, and we shouldn't care. If this were just created for the storybook example then please don't export them here so devs don't have access to them and move them directly to the storybook file.